classes/class-wc-connect-settings-view.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Settings_View' ) ) {
+
+	class WC_Connect_Settings_View {
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/488/commits/8ebe257dc446a63b65a6a909757e8a51ee1eb35a